### PR TITLE
#153728034 Unresolved checks dashboard

### PR DESCRIPTION
**Pull request by EleisonC**
**what does this PR do ?**
This PR allows a user to have a separate dashboard for unresolved checks

**Description of task to be completed**
Apart from the email notifications, the user should also have a separate dashboard view of their failed jobs

**How should this be manually tested?**
Clone and run the app locally then run the tests using the following command:
`./manage.py test` 
or clone and run the web app by following the instruction indicated in the readme.md file 

**What are the relevant pivotal tracker stories?**

[153728034](https://www.pivotaltracker.com/story/show/153728034)
**Screenshots**
screen shot 2018-01-17 at 07 55 53

### CODE REVIEW

File: hc/settings.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 SECRET_KEY = "---"
 DEBUG = True
 ALLOWED_HOSTS = ['0.0.0.0', 'localhost',
-                 'hc-the-knight.herokuapp.com']
+                 'hc-the-knight.herokuapp.com', '127.0.0.1']
```
Comment (mwaz) [inline hc/settings.py:23]: I think this change should have been made on a different branch @EleisonC 
This was what we were talking about changing general files that affect everyone causing conflicts.

**Review by mwaz — COMMENTED** [review]
Screenshots??

File: hc/settings.py (hunk 1 above)
Comment (mwaz) [reply hc/settings.py:23]: Kindly Add Screenshots to your PR

File: hc/front/tests/test_unresolved_checks.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from hc.api.models import Check
+from hc.test import BaseTestCase
+from datetime import timedelta as td
+
```
Comment (jimmykimani) [inline hc/front/tests/test_unresolved_checks.py]: remove the unnecessary white spaces

File: templates/front/unresolved_checks.html [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+                <button class="btn btn-danger btn-xs" data-toggle="button">{{ tag }}</button>
+            {% endif %}
+        {% endfor %}
+    </div>
+    {% endif %}
+
+</div>
+<div class="row">
+    <div class="col-sm-12">
+
```
Comment (jimmykimani) [inline templates/front/unresolved_checks.html]: same here avoid unnecessary line spaces

File: hc/front/tests/test_unresolved_checks.py (hunk 2 above)
Comment (EleisonC) [reply hc/front/tests/test_unresolved_checks.py]: on it

File: templates/front/unresolved_checks.html (hunk 3 above)
Comment (EleisonC) [reply templates/front/unresolved_checks.html]: on it

**Comment by mwaz** [comment]
👍

**Comment by jimmykimani** [comment]
Good work on your task but there's a minor issue. I wouldn't want to see my failed checks on my main dashboard I mean that's why we having a separate dashboard to view all failed jobs

**Comment by jimmykimani** [comment]
LGTM

**Review by jimmykimani — APPROVED** [review]

**Pull request merged**